draft-packages/menu/docs/MenuExamples.stories.tsx

```diff
@@ -0,0 +1,342 @@
+import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
+import duplicateIcon from "@kaizen/component-library/icons/duplicate.icon.svg"
+import editIcon from "@kaizen/component-library/icons/edit.icon.svg"
+import trashIcon from "@kaizen/component-library/icons/trash.icon.svg"
+import kebabIcon from "@kaizen/component-library/icons/kebab.icon.svg"
+import meatballsIcon from "@kaizen/component-library/icons/meatballs.icon.svg"
+import { withDesign } from "storybook-addon-designs"
+import React, { useState } from "react"
+import { figmaEmbed } from "../../../storybook/helpers"
+
+import { Menu, MenuList, MenuItem, StatelessMenu } from ".."
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+import { MenuContentExample } from "./components/MenuContentExample"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.menu}/Examples`,
+  component: Menu,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'import { Menu, MenuList, MenuItem, StatelessMenu } from "@kaizen/draft-menu";',
+      },
+    },
+    ...figmaEmbed(
+      "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=6262%3A1233"
+    ),
+  },
+  decorators: [withDesign],
+}
+
+export const DefaultStory = args => (
+  <Menu
+    {...args}
+    button={
+      <Button label={"Edit menu"} icon={chevronDown} iconPosition="end" />
+    }
+  >
+    <MenuList>
+      <MenuItem
+        onClick={e => {
+          e.preventDefault()
+          alert("onClick function to duplicate content")
+        }}
+        icon={duplicateIcon}
+        label="Duplicate item"
+      />
+      <MenuItem
+        onClick={e => {
+          e.preventDefault()
+          alert("onClick function to edit content")
+        }}
+        icon={editIcon}
+        label="Edit Item"
+      />
+      <MenuItem
+        onClick={e => {
+          e.preventDefault()
+          alert("onClick function to delete content")
+        }}
+        icon={trashIcon}
+        destructive
+        label="Delete item"
+      />
+    </MenuList>
+  </Menu>
+)
+
+DefaultStory.storyName = "Basic example"
+
+export const IconExample = () => (
+  <StoryWrapper>
+    <StoryWrapper.RowHeader headings={["Default", "Primary", "Secondary"]} />
+    <StoryWrapper.Row rowTitle="Variant">
+      <Menu button={<IconButton label={"Edit menu"} icon={meatballsIcon} />}>
+        <MenuContentExample />
+      </Menu>
+      <Menu
+        button={<IconButton primary label={"Edit menu"} icon={meatballsIcon} />}
+      >
+        <MenuContentExample />
+      </Menu>
+      <Menu
+        button={
+          <IconButton secondary label={"Edit menu"} icon={meatballsIcon} />
+        }
+      >
+        <MenuContentExample />
+      </Menu>
+    </StoryWrapper.Row>
+  </StoryWrapper>
+)
+
+IconExample.storyName = "Icon button menus"
```

Comment:
This might just be me, but I had to read "Icon button menus" 3 times before I could make sense of it.
To me "Menu using Icon Buttons" would be much clearer.

Reply:
Yeah that sounds much better 👍 Will update that